bin/i18n-map-extractor.js

```diff
@@ -138,7 +138,9 @@ class wpi18nExtractor {
 				chunkName = options.aliases.hasOwnProperty( chunk.name ) ?
 					options.aliases[ chunk.name ] :
 					chunk.name;
-				parseSourcesToMap( chunk._modules, chunkName, extractor );
+				if (chunk._modules) {
```

Comment:
I normally use following to check existence of a key in JS.

    if(typeof chunk._modules !== 'undefined')

Or since it's an object, you can use [hasOwnProperty](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty) method like the following:

    if(chunk.hasOwnProperty('_modules'))

Reply:
oh good call, ya a `hasOwnProperty()` check would be wise here since it's lower level JS

thnx, will fix